### ISSUE TITLE
Link descriptions from llcon.sourceforge.net to GitHub Wiki and new website

### ISSUE THREAD

**Issue by ann0see** [corrados/jamulus#581]
Google and some sites still link e.g. to http://llcon.sourceforge.net/sourcecodecompile.html, http://llcon.sourceforge.net/servermanual.html or similar. I think since maintaining two pages is too much work, I'd suggest to add a 301 Redirect (if possible) to the GitHub wiki sites or an info box states that more up to date information can be found on the GitHub wiki.

**Comment by corrados** [comment]
I can change the sourcecodecompile.html and servermanual.html, no problem. Can you create such a html page which does the redirect? I am no HTML expert...

**Comment by ann0see** [comment]
Maybe it can be done in HTMl but it wouldn't be the best way since the server wouldn't send the header. Probably it can be done via PHP or a .htaccess file (if the site allows .htaccess files which would be even better):
https://www.sistrix.de/frag-sistrix/onpage-optimierung/http-statuscode/3xx-redirection/wie-erstelle-ich-eine-301-weiterleitung/
I can help out if you want.
Furthermore a https secured page would be best: https://sourceforge.net/p/forge/documentation/Convert%20your%20website%20to%20HTTPS/
https://sourceforge.net/p/forge/documentation/Project%20Web%20Services/#https

**Comment by ann0see** [comment]
BTW: there are also other sites: http://llcon.sourceforge.net/examplesoundcard.html settingswindow.html etc. I'm unsure if they are all in the wiki but I think so. You can find them if you google `site:llcon.sourceforge.net`

**Comment by corrados** [comment]
I just found the following: https://sourceforge.net/p/forge/documentation/Project%20Web%20Services

They have a section for it: [Re-directing to Another Site](https://sourceforge.net/p/forge/documentation/Project%20Web%20Services/#re-directing-to-another-site)

Maybe we should just do what they recommend. What do you think?

**Comment by ann0see** [comment]
The PHP thing kind of works probably. But a 301 redirect tells google to update its index. If there's only the refresh tag I think it thinks that it's not a permanent redirect and therefore doesn't update it.

**Comment by corrados** [comment]
For a test I put only this `Redirect permanent /sourcecodecompile.html https://github.com/corrados/jamulus/wiki/Compiling` in the .htaccess and uploaded that file. But I still let the original `sourcecodecompile.html` file there (i.e. I have not deleted it yet). Do you think the redirect will still work even if the file is still there?

Unfortunately, it takes very long when I do changes to the web pages until they are finally apply. Last time with the banner update it took hours until I could see the change in my browser...

**Comment by ann0see** [comment]
For me it does the redirect perfectly.

**Comment by ann0see** [comment]
I'll look into it.

**Comment by ann0see** [comment]
`Redirect permanent /sourcecodecompile.html https://github.com/corrados/jamulus/wiki/Compiling
Redirect permanent /mainwindow.html https://github.com/corrados/jamulus/blob/master/src/res/homepage/manual.md#main-window
# I AM NOT SURE ABOUT THIS CHANGE:
Redirect permanent /servermanual.html https://github.com/corrados/jamulus/wiki/Running-a-Server
Redirect permanent /settingswindow.html https://github.com/corrados/jamulus/blob/master/src/res/homepage/manual.md#settings-window
Redirect permanent /requiredhardware.html https://github.com/corrados/jamulus/wiki/Hardware-Setup
Redirect permanent /examplesoundcard.html https://github.com/corrados/jamulus/wiki/Hardware-Setup#example-setups`

**Comment by gilgongo** [comment]
This is very good. The less fragmentation of the docs, the better 👍

**Comment by ann0see** [comment]
I think the website could and should link the wiki in a more prominent way. We could e.g. use a big button labeled "Get started now!". This is based on @gilgongo comment here https://github.com/corrados/jamulus/issues/534#issuecomment-680225729

The download button should be less prominent.

Also the site looks quite empty at the moment. I think we could add e.g. YouTube videos (we should use a opt in library here for privacy reasons but that’s not a problem)
I can look into these suggestions if you want.

**Comment by corrados** [comment]
> I hope I got everything right. Please have a look especially at the line with the server manual.

Looks good. I have just updated the file on the server. Please try out if it works for you, thanks.

**Comment by ann0see** [comment]
Yep. Looks good for me. Are there any other files? I've included all the files I could find on google.

**Comment by corrados** [comment]
I think you covered the important ones. That's fine.

> I think the website could and should link the wiki in a more prominent way.

Thank you for your proposal. But I think we may use a complete different design of that homepage. It does look a bit outdated. It seems you are a experienced web coder. If you like you can be even more creative :-).

**Comment by ann0see** [comment]
> It seems you are a experienced web coder.

I wouldn’t say that (especially design wise) but for sure I can look into it (if I find enough time)

We could also use a template from https://html5up.net/?

**Comment by ann0see** [comment]
![Screenshot_2020-09-12 Jamulus Software(1)](https://user-images.githubusercontent.com/20726856/92995798-feb36500-f506-11ea-963b-20f5761e86af.png)
First impressions.

**Comment by corrados** [comment]
Looks good. But you should avoid "With (almost) no delay". That could raise wrong expectations on the Jamulus software. Usually the people do not have optimized hardware setups so they will have significant delay (unfortunately...).
And also our new Banner is missing. This is a must have on the homepage :-).

@gilgongo Since you are the Wiki manager, what are your comments to the new design?

**Comment by ann0see** [comment]
About the banner: I know. It's quite difficult to add. If it is added at the top (which I would prefer) the button would have to be moved.
Maybe it can be added under "Help needed?"

**Comment by corrados** [comment]
I cannot answer you that. A graphic designer should comment on that.

**Comment by ann0see** [comment]
Maybe this could work:
![Screenshot_2020-09-12 Jamulus Software(3)](https://user-images.githubusercontent.com/20726856/92998653-46dc8280-f51b-11ea-98d6-3b7eb8fe5937.png)

**Comment by ann0see** [comment]
Do you have a SourceForge Logo without background?

**Comment by corrados** [comment]
> Is this the official one?

Yes, it is.

>  Do you have a SourceForge Logo without background?

This is what they have:
[![Download Jamulus - Internet Jam Session Software](https://sourceforge.net/sflogo.php?type=11&group_id=158367)](https://sourceforge.net/p/llcon/)
[![Download Jamulus - Internet Jam Session Software](https://sourceforge.net/sflogo.php?type=13&group_id=158367)](https://sourceforge.net/p/llcon/)
[![Download Jamulus - Internet Jam Session Software](https://sourceforge.net/sflogo.php?type=14&group_id=158367)](https://sourceforge.net/p/llcon/)
[![Download Jamulus - Internet Jam Session Software](https://sourceforge.net/sflogo.php?type=16&group_id=158367)](https://sourceforge.net/p/llcon/)

**Comment by ann0see** [comment]
Unfortunately they all have a background. Therefore the link will look quite odd in the footer.
@gilgongo Should we link the direct download button or the wiki in a prominent way. I vote for the wiki.

**Comment by ann0see** [comment]
Latest version. There's still something wrong with the layout (something seems to be bigger than the width of the screen) but I couldn't find the problem. 
![Screenshot_2020-09-12 Jamulus Software(6)](https://user-images.githubusercontent.com/20726856/93005634-16b0d600-f553-11ea-8a4e-0dc3088c4576.png)
I've added a [humans.txt](http://humanstxt.org/) file.

**Comment by gilgongo** [comment]
> Since you are the Wiki manager, what are your comments to the new design?

I would say that we would be best served by brevity on this page. I'm a bit nervous about trying to summarise all the aspects of Jamulus (how it works, what libraries it uses, what you need to use it, etc.) in case people think that's the totality of the information we have (which would be the expectation if you landed on a page like this and saw that amount of information).

Instead, the idea of simply saying something very compact and then taking them to the wiki would be safer.

So simply this I think (with the footer - sorry, I'm on a chromebook with no proper image editor):

![jamuilushome](https://user-images.githubusercontent.com/4561747/93014752-9e2a3380-f5ab-11ea-8de2-9798080414be.png)


I would tweak the strapline slightly though from "Make music. Online. With (almost) no delay" to:

"Play music online with friends. For free."

The reason is that (in English) "make music" implies Jamulus is a too to create the music itself. I wouldalso emphasise the social nature of it ("with friends") and avoid hinting at complications ("almost no delay") at this stage and instead emphasise the free aspect of it.

Possibly also say "or download now" instead of "download Jamulus directly" as that's a bit more natural.

BTW if we are concerned with SEO, we could provide a summary text panel but need to craft that so that it doesn't look like that's all we have. Perhaps something like:

"Jamulus is for playing, rehearsing, or just jamming with your friends, your band or just anyone you find online. Use your Windows, Mac or Linux machine to connect to Jamulus servers worldwide, for free over a normal broadband connection. Or host your own private server. Jamulus is designed for high quality, low-latency sound, making it easy to play together remotely and in time. Have a look at how people do it [link here to the demos page]"

As to the graphical design, the contrast between the blue and the white is quite low and I worry about some people (or some monitors) not being able to read it very easily. 

Finally, I (and some others) had another thought about how to use the main "cloud" design, but we should talk to the author about that. I'm afraid I've forgotten who that is though.

**Comment by ann0see** [comment]
@gilgongo Fair points.

1. I would say that we would be best served by brevity on this page.
I agree. The site should be something like a [landing page](https://en.wikipedia.org/wiki/Landing_page) in my opinion it should also include text (like you suggested).
Libraries etc. are unneeded. Maybe that it's licensed under GPL is important.
2. "Play music online with friends. For free."
That's quite long but works probably. 
3. or download now
It should be clear that this link shouldn't be clicked by average users.
Maybe we could say "Advanced: download now"
4. contrast
The contrast was one of the main issues. If you make the background darker the contrast between logo and background would be bad. If you want, you can try to change the color (https://github.com/ann0see/jamulus_website/blob/a2ce7a623cbe13de6b0ee5d54508e2bb067e4227/newdesign/assets/css/custom.css#L6) and open a PR on my repo.
I came up with the color palette by using this site https://imagecolorpicker.com/
5. thought about how to use the main "cloud" 
It should be in the git history. @geheimerEichkater?

**Comment by gilgongo** [comment]
Yes, "landing page" is a good description.

Re. 2: how about just "Play music with friends. For free." Shorter? I only really added "online" for an SEO keyword scent. 

Re. 3: I'd say not have it then really, Oh BTW I assume the "get started" button [goes here](https://github.com/corrados/jamulus/wiki/Getting-Started), yes? I which case there's a download link on every page from then on.

re. 5: Yes, separate ticket for that stuff I think.


(Note to myself: always remember to number your points)

**Comment by ann0see** [comment]
1: In SEO: never design for search engines ;-). We can remove "How does Jamulus work".

2: I think we can leave the .."online".... It looks ok.

3: Yes. The button goes there. I'd add the download link just in case somebody already knows Jamulus and searches the download. We could check which link gets clicked on more often (e.g. via a link shortener like is.gd bit.ly short url etc.) We can then afterwards decide how to continue

4: I'm unsure about the contrast. Probably there's a way to "fix" the problem. But as I said, I am not a designer.

5: It would be great if he looked here ;-) but a separate ticket is great.

**Comment by gilgongo** [comment]
Agree with 1., although the main thing I'm worried about is that people will come to a seemingly "fully featured" page and (subconsciously or otherwise) think "Oh, all the info is here. OK I'll just download it and get going then." The more people we can get going through the wiki to gain a better understanding before install, the better. 

The alternative is to make this a place that **does** have access to all the information, and we take the decision to move from using the wiki to using the SF platform for a properly designed, multi-lingual site. eg for https://github.com/corrados/jamulus/issues/77#issuecomment-691755254

Fragmentation of information sources about Jamulus on the web is something we need to avoid, in my opinion.

**Comment by ann0see** [comment]
My new proposal has quite a lot removed now. Only the help section and "Jamulus is open source" is included now. Does this still evoke a "complete" feeling?

Translation would be great but is a huge amount of work: we‘d need a CMS like WordPress, Contao, Typo3 etc. for the site or at least some kind of "intelligent server" which gets the browser language (could be done with PHP probably).
Concerning a translation of the wiki I wouldn’t include it directly in the site but use external tools.

**Comment by gilgongo** [comment]
The wiki is being re-configured for translation though (although I'm a bit unclear on procedure going forward)

**Comment by gilgongo** [comment]
> My new proposal has quite a lot removed now.

Sorry, where is this? Here?  https://github.com/ann0see/jamulus_website

**Comment by ann0see** [comment]
Yes. It’s there

**Comment by ann0see** [comment]
You can see the site on https://ann0see.github.io/jamulus_website/newdesign/ now. It's currently blocked by a robots.txt from google

**Comment by gilgongo** [comment]
Yes - that looks good for the information architecture 👍 

The issue of the graphic design (logos, contrast) we can look at later if @geheimerEichkater is around.

**Comment by ann0see** [comment]
Ok. @corrados should it be published (with a link including click statistics)?

**Comment by corrados** [comment]
Yes, I can do it on the weekend.

> with a link including click statistics

I don't like click statistics, actually. I would prefer to do it without it.

Comments to the page:

- Why do we have to Get Started Now button twice? I think one is enough
- Have you tried to exchange the logo and the banner? How does this look like.

**Comment by gilgongo** [comment]
> Why do we have to Get Started Now button twice? 

I am assuming on most screens you'd only see it once in the viewport, so it's there when you have scrolled down.

**Comment by ann0see** [comment]
> > Why do we have to Get Started Now button twice?
> 
> I am assuming on most screens you'd only see it once in the viewport, so it's there when you have scrolled down.

Exactly. Also people should see a button if they read everything

**Comment by corrados** [comment]
> Once you upload the site, please use https (for security reasons and SEO)

Do you know how to do it?

**Comment by corrados** [comment]
> The changes are live and on the logoswap branch. I don't really like it but I can live with it.

I agree with you. Maybe we should use the old design with the logos swapped as it was before. @gilgongo What is your opinion?

**Comment by geheimerEichkater** [comment]
Hello and sorry for being late to the party.

The current state looks very nice so far, I'd be happy to provide the following assets for minor design tweaks: 
- provide a photo background (faint clouds) for the whole width of the light blue header/hero 
- provide a solitary white transparent "Jamulus Logo" (the wide cloud with the typography underneath) to sit responsively on top (svg or png)
- the CtA "Get started now!" could use a white border to better unite visually with the over all design language, making it basically appear like a ghost-button (I'll attach a mockup to this post) OR/AND highlight the button on top with a bold contrasting bg-color to draw more attention. Obvious choices would be orange as direct contrast or maybe greenish, or a red tint, or yellow … I would try out and suggest a couple of choices, if so desired.

![mockup-landingpage](https://user-images.githubusercontent.com/67019343/93616313-df1faf00-f9d4-11ea-8870-22717c9ae81d.png)

**Comment by ann0see** [comment]
@corrados it should be on the site you mentioned before: https://sourceforge.net/p/forge/documentation/Project%20Web%20Services/#https

> I agree with you. Maybe we should use the old design with the logos swapped as it was before. @gilgongo What is your opinion?

What about @geheimerEichkater s' suggestion? 

I'd prefer SVG and/or plain CSS for the background.
The cloud should be SVG.

I can make the button a ghost button.

**Comment by gilgongo** [comment]
> > Once you upload the site, please use https (for security reasons and SEO)
> 
> Do you know how to do it?

@corrados 

https://sourceforge.net/p/llcon/admin/ext/https/

I'm too scared to click that button :-)

**Comment by ann0see** [comment]
This has to be tested with the new site (you don’t want mixed content which would happen since the sourceforge logo is currently loaded via http

**Comment by gilgongo** [comment]
@ann0see The SF guidance says you should change any absolute links to root-relative ones unless they're 3rd party.

**Comment by corrados** [comment]
@gilgongo I gave you admin rights on the Sourceforge Jamulus project. Can you please take care of the upload of the new homepage?

Note: If you change to https, make sure that the link to the paper on https://github.com/corrados/jamulus/wiki still works.

**Comment by ann0see** [comment]
Before uploading please inform me since I have to minify the CSS and HTML before.

**Comment by gilgongo** [comment]
@ann0see @corrados OK I've uploaded that and turned on https so the new URL should be https://llcon.sourceforge.io. Maybe some delay or cache issue means that's redirecting back to the http for me, but we'll see in a bit. The paths looks OK meanwhile I think.

BTW I added the existing .htaccess file just in case out of date incoming links still need redirecting.

Now I just need to fix the link to Volker's paper. I put it in the docroot I thought, but maybe I made a mistake.

**Comment by ann0see** [comment]
@gilgongo can you take a Screenshot?

**Comment by gilgongo** [comment]
I'm on a Chromebook:

![Screenshot 2020-09-19 at 12 38 18 PM](https://user-images.githubusercontent.com/4561747/93666332-4725d200-fa75-11ea-98f5-6901e79fe1d0.png)

**Comment by ann0see** [comment]
Thanks! Should be fixed now

**Comment by gilgongo** [comment]
OK I re-uploaded your "upload" directory. Might have to wait a bit for caches to clear maybe? It's still not doing the https either...

**Comment by ann0see** [comment]
Are you sure that https was correctly enabled? Maybe try to set it again?

**Comment by gilgongo** [comment]
It's just a button that you click:

![Screenshot 2020-09-19 at 3 52 13 PM](https://user-images.githubusercontent.com/4561747/93670059-36cf2080-fa90-11ea-8b0f-5726ad80a83b.png)

**Comment by gilgongo** [comment]
Oh BTW SF have a transparent button we can put in the footer that would look better than the current white one:

~~~
<a href="https://sourceforge.net/p/llcon/"><img alt="Download Jamulus - Internet Jam Session Software" src="https://sourceforge.net/sflogo.php?type=17&group_id=158367" width=150></a>
~~~

**Comment by ann0see** [comment]
That’s good news. I‘ll look into it.

**Comment by ann0see** [comment]
Ok. I've updated it. Unfortunately the logo isn't white (and probably copyrighted). I've added a white background and fixed some layout issues on mobile devices.

**Comment by ann0see** [comment]
Tested the site again and https still doesn’t work. Can you disable it, wait 5 minutes and enable it again?

If that doesn’t work, we should notify sourceforge support.

**Comment by gilgongo** [comment]
OK that logo looks better. I've switched it back to http now and will wait a bit before going to https again.

BTW the contrast on the white text against the blue background fails WCAG AA. What do you think about making it dark grey text instead? A grey keeps it feeling "cloudy":

https://webaim.org/resources/contrastchecker/?fcolor=FFFFFF&bcolor=8FC8DD

vs

https://webaim.org/resources/contrastchecker/?fcolor=3B3B3B&bcolor=8FC8DD

**Comment by ann0see** [comment]
Can you open a PR on my repo (change it in upload/assets/css/custom.css and newdesign/Assets/css/c.min.css

I‘ll probably not be able to change it for the next few hours

**Comment by gilgongo** [comment]
Hm. I can't see those files (did you mean /upload/assets/css/c.min.css and newdesign/assets/css/custom.css?). I'll leave it for now.

**Comment by geheimerEichkater** [comment]
One other suggestion: the logo icon (three bubbles cloud with the flash) also exits in an inverted version, exactly for that case. This way, you exchange the logo, use the #49a9c9 blue for the whole header/hero bg and then the contrast should be sufficient. See preview in attachment.

![icon-inv-neg](https://user-images.githubusercontent.com/67019343/93706640-72b5c480-fb28-11ea-8956-cc631106218a.png)

**Comment by gilgongo** [comment]
That might be better (particularly with the ghost button white border), but the text will have to be darker if the background is darker. White text on #49a9c9 still doesn't have enough contrast.

**Comment by ann0see** [comment]
@gilgongo Yes. Sorry I meant what you mentioned.
@geheimerEichkater Could you please send a SVG version? I can try that.

**Comment by gilgongo** [comment]
Support ticket on SF for the https thing: https://sourceforge.net/p/forge/site-support/21270/

**Comment by ann0see** [comment]
> One other suggestion: the logo icon (three bubbles cloud with the flash) also exits in an inverted version, exactly for that case. This way, you exchange the logo, use the #49a9c9 blue for the whole header/hero bg and then the contrast should be sufficient. See preview in attachment.

Ok. Done!

**Comment by geheimerEichkater** [comment]
> I've edited and minified the SVG. I hope that's ok for you?
Thanks, yes alright, matter of fact … I cannot attach svg to this conversation. The background of the header could now be adapted to the logo-blue to contrast a little better against the white typo than the current light blue does.

> Nevertheless, I'd probably prefer your suggestion with the cloud.
Do you mean the suggested photographic element? I would consider @gilgongo advice and if so, then provide the bg-graphic with an even darker blue, just dark enough to validate contrast against white typo. Anybody else in favor of the photographic element? Should I go for it?

**Comment by ann0see** [comment]
I’m quite sure I did that already:

https://ann0see.github.io/jamulus_website/upload/

@gi

**Comment by ann0see** [comment]
Background color seems to be fixed. 

The SourceForge Support answer isn’t very helpful. There are no http links on that site

**Comment by gilgongo** [comment]
They've got https up now.

So are we all done now?

**Comment by ann0see** [comment]
I‘d say yes, we’re finished for the first version. There can still be some improvements (@geheimerEichkaters‘ suggestion with the cloud, translatable page,... but that’s all future work. I‘ll close the issue now.)

**Comment by ann0see** [comment]
#615 

@corrados please undo the .htaccess changed and reupliad the old files for SEO. We don‘t want to loose reputation.

**Comment by corrados** [comment]
@gilgongo Can you please take care of all these Jamulus homepage issues/required changes?

**Comment by gilgongo** [comment]
> please undo the .htaccess changes and re-upload the old files for SEO. We don‘t want to loose reputation.

Not sure I understand. The .htaccess file currently redirects old URLs to new ones. So do you mean the redirects weren't what we wanted and that we should still have those URLs working as they were?

BTW I removed the old files (I have a backup) apart from Volker's .pdf and the .htaccess, and replaced them with the new ones.

**Comment by ann0see** [comment]
The problem is that the GitHub wiki is not indexed —> google might remove the links and it’s reputation.

**Comment by gilgongo** [comment]
OK (I assume there's no duplication in the file names).

**Comment by ann0see** [comment]
No. We shouldn't change anything yet (that means the old pages except for the homepage should be uploaded) until #615 is solved.

**Comment by gilgongo** [comment]
I sense the stakes are not that high though :-) A search for "internet live music playing" shows Jamulus not ranking at all as far as I can see. Or has that just happened this week?

**Comment by gilgongo** [comment]
OK that's done. All the old files are back up (but no .htaccess and index.html re-named index-old.html)

**Comment by ann0see** [comment]
That’s good. Now we can continue in the other issue

**Comment by ann0see** [comment]
https://llcon.sourceforge.io/servermanual.html

Ranks quite high (search for setup jamulus).

This page should now show a text saying: We are currently migrating to an other wiki software. More up to date information during transmission period can be seen on the GitHub wiki.

Client setup: <link>

Server setup: <link>

**Comment by gilgongo** [comment]
> (search for setup jamulus)

Well, yes :-) "jamulus" is a unique word so I would hope at least the home page (although we have several) would come in as number 1.  But if you don't know about Jamulus, then you're not going to type that in to Google.

**Comment by ann0see** [comment]
I've added a big red box to the servermanual.html file. It can be seen in my repo. You just need to upload it. I know it's quite hacky (added a <style> element but it's faster). We could consider a temporary redirect but I'd first solve the potential wiki change.

**Comment by gilgongo** [comment]
OK done that - looks great! :-)

**Comment by WolfganP** [comment]
> I've added a big red box to the servermanual.html file. It can be seen in my repo. You just need to upload it. I know it's quite hacky (added a <style> element but it's faster). We could consider a temporary redirect but I'd first solve the potential wiki change.

Just a quick misspelling correction on the red box, "an other" should be "another"

**Comment by ann0see** [comment]
Fixed.

**Comment by ann0see** [comment]
@gilgongo I just thought again and would say that a Redirect 302 instead of Redirect permanent could also work. 302 tells google to redirect but not delete old entries. This might solve the problem for now until the other issue is solved. I hope you still have the .htaccess

**Comment by gilgongo** [comment]
Oh - I deleted the old .htaccess. Do you not have it somewhere?

**Comment by ann0see** [comment]
No, unfortunately not. But that's no issue. 

Could you please make a screenshot of the document root of the page (so I can see all the names of the html pages?)

If you want to do it on your own, you can follow https://www.hostwinds.com/guide/set-temporary-302-redirect-using-htaccess/

**Comment by gilgongo** [comment]
This is what's currently there:

~~~
android-icon-144x144.png  apple-icon-60x60.png        favicon.ico        Jamulus-version-installer.exe                         __servermanual.html
android-icon-192x192.png  apple-icon-72x72.png        humans.txt         __mainwindow.html                                     _servermanual.html
android-icon-36x36.png    apple-icon-76x76.png        images             _mainwindow.html                                      servermanual.html
android-icon-48x48.png    apple-icon.png              _index.htm         mainwindow.html                                       __settingswindow.html
android-icon-72x72.png    apple-icon-precomposed.png  _______index.html  manifest.json                                         _settingswindow.html
android-icon-96x96.png    assets                      ______index.html   ms-icon-144x144.png                                   settingswindow.html
apple-icon-114x114.png    browserconfig.xml           _____index.html    ms-icon-150x150.png                                   ____sourcecodecompile.html
apple-icon-120x120.png    __examplesoundcard.html     ____index.html     ms-icon-310x310.png                                   ___sourcecodecompile.html
apple-icon-144x144.png    examplesoundcard.html       ___index.html      ms-icon-70x70.png                                     __sourcecodecompile.html
apple-icon-152x152.png    favicon-16x16.png           __index.html       PerformingBandRehearsalsontheInternetWithJamulus.pdf  sourcecodecompile.html
apple-icon-180x180.png    favicon-32x32.png           index.html         __requiredhardware.html                               style.css
apple-icon-57x57.png      favicon-96x96.png           index-old.html     requiredhardware.html

~~~

**Comment by ann0see** [comment]
Ok. The .htaccess could have a content like:

```
Redirect 302 /examplesoundcard.html https://github.com/corrados/jamulus/wiki/Hardware-Setup#example-setups
Redirect 302 /settingswindow.html https://github.com/corrados/jamulus/blob/master/src/res/homepage/manual.md#settings-window
Redirect 302 /mainwindow.html https://github.com/corrados/jamulus/blob/master/src/res/homepage/manual.md#main-window
Redirect 302 /requiredhardware.html https://github.com/corrados/jamulus/wiki/Hardware-Setup
Redirect 302 /sourcecodecompile.html https://github.com/corrados/jamulus/wiki/Compiling
```
The other  html files can be moved to a folder e.g. named old:

```
__examplesoundcard.html
_index.htm
_______index.html
______index.html
_____index.html
____index.html
___index.html
__index.html
index-old.html --> rename to e.g. index-2020-08.html
__mainwindow.html
_mainwindow.html
__requiredhardware.html
__servermanual.html
_servermanual.html
__settingswindow.html
_settingswindow.html
____sourcecodecompile.html
___sourcecodecompile.html
__sourcecodecompile.html
style.css
```

What's Jamulus-version-installer.exe? Probably an old jamulus version?

**Comment by corrados** [comment]
I sometimes use the sourceforge web directory for sharing beta/test versions of Jamulus. You can remove this one now but I may add some files later on in that directory. But that should not hurt.

**Comment by ann0see** [comment]
Ok. If I were you, I'd create a folder e.g. share/ or upload/ to share files (to have the document root organized and clean)
